e2e/pages/resources/cms-lookup/cms-lookup.readonly.spec.js

```diff
@@ -29,11 +29,11 @@ describe('On cms reverse look up page', () => {
             it('should show correct listings for the CMS ID', () => {
                 browser.waitUntil( () => $(cmsLookup.lookupResultsTable).isDisplayed());
                 var ls = [];
-                var length = $$('//*[@id="lookupCertIdResults"]/tbody/tr').length;
+                var length = cmsLookup.rowsLookupResultsTable.length;
                 for ( var j = 1; j <= length; j++ ) {
                     ls.push($('//*[@id="lookupCertIdResults"]/tbody/tr[' + j + ']/td[6]').getText());
```

Comment:
This should be a locator too. I've done things in the Product edit where I pass, to the Page Object file, an element, and have that file return some sub-element from that element. Something like 
```suggestion
                    ls.push(page.getChplProductNumber(cmsLookup.rowsLookupResultsTable(j)).getText());
```

Reply:
Not sure if it will work in this case as its table, but i did what i think will be fine here.